src/references/interaction_system.md

```diff
@@ -0,0 +1,154 @@
+
+<!-- TODO: Check that link -->
+
+## Adding the Interaction System to a Project
+
+### Add the Interaction System Package
+
+1. In the **Core Content** window, open the **Game Components** section under **GAME OBJECTS**.
+2. Open the **Interaction** category to find the **Interaction System Package** template.
+3. Drag the **Interaction System Template** into the scene or **Hierarchy** to add the scripts and examples to the project.
+
+### Access the Interaction System Modules and Components
+
+In the **Project Content** window left-side navigation, open the **My Scripts** section and open the **Interaction System** folder that was added with the package.
+
+- The **Components** folder contains examples of interactions which can be used as a way to create interaction without scripting.
+- The **Examples** folder contains dependencies for the examples. The examples themselves can be found in the **My Templates** section, detailed below.
+- The **Modules** folder contains the four different interaction modules.
+
+!!! warning
+    The **Interaction System** creates folders both in **My Scripts** and **My Templates**. The **My Templates** section contains the examples, and all the components and modules of the Interaction System are available in **My Scripts**.
+
+### Find the Examples
+
+Once the **Interaction System Package** has been added to your project, all the examples and component scripts can be found in the **Project Content** window.
+
+1. Open the **Project Content** window and find the **My Templates** section in the left-side navigation.
+2. Open the **Interaction System** folder in **My Templates**.
+3. Select the **Examples** folder to find the four interaction examples.
+4. Drag one of the examples into the game and press **Play** ![Play](../img/EditorManual/icons/Icon_Play.png) to test the interaction.
+{ .image-inline-text .image-background }
+
+!!! note
+    For all the examples, ``ability_primary`` is the default keybinding, and are therefore activated by clicking. This can be changed by changing the **PressBinding** property on the component scripts.
+
+## Interaction System Modules
+
+### Cursor Interaction Module
+
+The **Cursor Interaction Module** allows custom interactions with UI Buttons, including selecting multiple elements.
+
+#### Example
+
+- Hovering the mouse over the ``?`` icon reveals a tooltip using the **Toggle Tooltip Component**.
+- Clicking and dragging on the outer part of the window will reposition it using the **Draggable Object Component**.
+- Each of the white boxes will change color when clicked using the **Change Color Component**, and add themselves to a selection set using the **Selectable Component**.
+
+<!-- TODO: Add video of interaction example -->
+
+### Look
+
+The **Look Interaction Example** checks if a player's reticle (or the center of the screen, regardless if there is a displayed reticle) is intersecting a specific object.
```

Comment:
The Look module actually uses the Player look direction. By default that is the center of the screen, but if you move the camera away from its starting position the look direction will no longer be centered and instead a ray from the Avatars head to a point in front of it.

You can see this ray if you add a "Interaction Settings" template to the scene and check "ShowDebug". You can also use these settings to adjust other aspects of the system (like reduce the length of that ray)